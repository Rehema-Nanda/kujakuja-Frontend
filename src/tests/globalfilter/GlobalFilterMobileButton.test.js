import React from "react";
import renderer from "react-test-renderer";
import GlobalFilterMobileButton from "../../components/globalfilter/GobalFilterMobileButton";

it("renders correctly", () => {
    const props = {
        text: "Test",
        onClick: jest.fn(),
    };

    const tree = renderer.create(
        <GlobalFilterMobileButton {...props} />,
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
