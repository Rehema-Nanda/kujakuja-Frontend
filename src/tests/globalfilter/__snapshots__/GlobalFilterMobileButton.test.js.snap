// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  aria-label={null}
  className="globalfilter-mobile-button btn btn-secondary"
  onClick={[Function]}
  type="button"
>
  Test
  <div
    className="globalfilter-button-caret"
  >
    <i
      className="fa"
    />
  </div>
</button>
`;
